src/apps/competitions/models.py

```diff
@@ -28,6 +29,8 @@ class Competition(ChaHubSaveMixin, models.Model):
     description = models.TextField(null=True, blank=True)
     docker_image = models.CharField(max_length=128, default="codalab/codalab-legacy:py3")
 
+    queue = models.ForeignKey(Queue, on_delete=models.SET_NULL, null=True, blank=True, related_name='competitions')
```

Comment:
How does this value get changed? I don't see anything additions to the comp editor?

Reply:
Added to comp editor